examples/all_plugins/pubspec.yaml

```diff
@@ -1,72 +1,81 @@
+### Generated file. Do not edit. Run `pub global run flutter_plugin_tools gen-pubspec` to update.
 name: all_plugins
 description: Flutter app containing all 1st party plugins.
 
-# The following defines the version and build number for your application.
-# A version number is three numbers separated by dots, like 1.2.43
-# followed by an optional build number separated by a +.
-# Both the version and the builder number may be overridden in flutter
-# build by specifying --build-name and --build-number, respectively.
-# In Android, build-name is used as versionName while build-number used as versionCode.
-# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
-# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
-# Read more about iOS versioning at
-# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ^2.0.0
 
 dependencies:
-  flutter:
+  flutter: 
     sdk: flutter
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^0.1.2
+  android_alarm_manager: 
+    path: /Users/bmparr/Work/plugins/packages/android_alarm_manager
+  android_intent: 
+    path: /Users/bmparr/Work/plugins/packages/android_intent
```

Comment:
I don't think you want to check in these local paths. Maybe revert and add pubspec.yaml to .gitignore or something.